analysers/experiments/cpu.py

```diff
@@ -181,6 +181,12 @@ def sortAndGet(field, asc):
 CassandraRDD = sc.cassandraTable(cassandraKeyspace, "trial_cpu_core")
 CassandraRDD.cache()
 
+nOfCores = CassandraRDD.select("cpu_mean") \
```

Comment:
@VincenzoFerme  There we have the active CPUs, but I want the total number of CPUs